packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -509,19 +511,23 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
     };
     /* @conditional-compile-remove(close-captions) */
     const onStartCaptions = async (options?: CaptionsOptions): Promise<void> => {
-      await call?.feature(Features.TeamsCaptions).startCaptions(options);
+      const captionsFeature = call?.feature(Features.Captions) as unknown as TeamsCaptions;
```

Comment:
Why do we have all these captions changes in here? @carocao-msft should these be a part of this or should this PR be merging into your captions PR?